tfmigrate/test_xmv_action_test.go

```diff
@@ -0,0 +1,165 @@
+package tfmigrate
+
+import (
+	"context"
+	"testing"
+
+	"github.com/minamijoyo/tfmigrate/tfexec"
+)
+
+func TestAccStateMvActionWildcardRefactorToModule(t *testing.T) {
+	tfexec.SkipUnlessAcceptanceTestEnabled(t)
+
+	backend := tfexec.GetTestAccBackendS3Config(t.Name())
+
+	source := `
+resource "aws_security_group" "foo" {}
+resource "aws_security_group" "bar" {}
+resource "aws_security_group" "baz" {}
+`
+	tf := tfexec.SetupTestAccWithApply(t, "default", backend+source)
+	ctx := context.Background()
+
+	updatedSource := `
+module "sg_module" {
+ source = "./sg_module"
+ for_each = toset(["foo", "bar", "baz"])
```

Comment:
>      The name "for_each" is reserved for use in a future version of Terraform.

The test failed in Terraform v0.12.31 because the module for_each was introduced in Terraform v0.13.
If we verify advanced examples such as module for_each as a unit test of getStateMvActionsForStateList, the acceptance test could be simpler, like the example in README.

Reply:
OK, understood, I'll swap out the acceptance test out with the case from the README.md and will have this acceptance test covered as a unit test.